scripts/benv

```diff
@@ -139,12 +139,12 @@ function setFrequency() {
 
   local freqs=$(cat /proc/cpuinfo | grep MHz)
   local c=$(echo "$freqs" | grep $cpuFrequency.000 | wc -l)
-  [[ "$c" == "4" ]] ||
+  [[ "$c" != "4" ]] ||
```

Comment:
I think it was correct before; if `c` is 4, we seen all 4 cpus at the correct frequency, so `||` will not execute `err`. Or am I missing something?

Reply:
You're right Lukas, there was another error in the code (that I modified) that automatically made me think this was the line causing me problems. I didn't even think about what I was modifying, I was in trial and error mode :smile:. In my case, this condition is almost always false because I use intel_pstate and it reports results that vary by small digits of Hz, I guess you don't see that because you use `acpi_cpufreq`.

Reply:
yes, we disabled `intel_pstate`. see some notes here: https://github.com/scala/scala-dev/issues/338